test/lib/tpunit++.hpp

```diff
@@ -305,21 +308,27 @@ namespace tpunit {
                                          const std::list<std::string>& before, const std::list<std::string>& after, int threads,
                                          std::function<void()> threadInitFunction);
 
+         /**
+          * This method writes to a temporary buffer and formats the message nicely for debugging
+          * purposes if a test were to fail.
+          */
+         void testLog(const string& newLog);
```

Comment:
What if we called this `TESTINFO` or something that looks more like our normal logging functions?

Reply:
Sure!